### PR TITLE
Add options and functionality for skipping receptor animations

### PR DESCRIPTION
**Pull request by Zageron**
Only remaining issue is overflow. Please review

### CODE REVIEW

**Comment by Zageron** [comment]
Closes #289

File: src/classes/Language.as [hunk 1]
```diff
@@ -196,6 +196,10 @@ package classes
                 }
                 indexed[data[lang]["_index"]] = lang;
             }
+
+            data["us"]["options_receptor_animations"] = "Show Receptors Animations";
+            data["us"]["options_receptor_speed"] = "Receptor Flash Speed:";
+
```
Comment (Sploder12) [inline src/classes/Language.as:202]: These should probably be strings from the language database

**Review by Sploder12 — DISMISSED** [review]
Looks good!

File: src/classes/Language.as (hunk 1 above)
Comment (Zageron) [reply src/classes/Language.as:202]: Will do

**Comment by Zageron** [comment]
image

**Review by Sploder12 — APPROVED** [review]
Looks beautiful!

**Pull request merged**